src/components/Dropdown/Dropdown.stories.tsx

```diff
@@ -236,9 +241,9 @@ export const UsingWithAnotherComponent: Story<DropdownProps> = () => (
     <CreatableSelect
       createNewLabel="Buy new kitten"
       options={options}
-      defaultIsMenuOpen
       isMenuPositionRelative
       isMulti
+      menuIsOpen
```

Comment:
I don't see the corresponding change to this prop. May have missed it but worth checking it out.

Reply:
This is property from the `react-select`, it's supported, but it's not documented (thus use without any warranty)